### ISSUE TITLE
Confusing colon in some subject chains

### ISSUE THREAD

**Issue by acka47** [hbz/lobid#307]
Working on https://github.com/hbz/nwbib/issues/317 @fsteeg and me noticed some strange subject chains with a leading `:` or with subjects starting with a `:`.

Examples:

http://nwbib.de/HT015854197
http://nwbib.de/HT018529018

The colons are added in front of the content from subfield `t`. E.g. the [source of HT018529018](http://lobid.org/hbz01/HT018529018) includes this:

``` xml
<datafield tag="902" ind1="-" ind2="1">
  <subfield code="t">Antikensammlungen Ursula und Karl-Heinz Preuß</subfield>
  <subfield code="9">(DE-588)1062588460</subfield>
</datafield>
```

The relevant part in the current morph is [lines 35-45](https://github.com/lobid/lodmill/blob/e4971d5b4b50e6115189e4bede7de01deb8ac6a9/lodmill-rd/src/main/resources/morph-hbz01-to-lobid.xml#L39-L45).

**Comment by dr0i** [comment]
If this is about to change the old data I would suggest to open an issue at the proper repo.

For the API2-0 data, there was an issue with the label which is fixed with #426, see e.g. http://lobid.org/resources/HT015854197 vs. http://test.lobid.org/resources/HT015854197.

**Comment by acka47** [comment]
Moving this in the backlog in the same repo. Hopefully, this will be fixed when we will move to API 2.0 at some point...

**Comment by acka47** [comment]
In API 2.0 we donÄt have and don't need `subjectChain` anymore. Closing.
